clarity/contracts/pool/collateral-rebalancing-pool.clar

```diff
@@ -579,16 +582,16 @@
                 (weight-x (- ONE_8 weight-y))            
             
                 ;; fee = dx * fee-rate-x
-                (fee (mul-up dx fee-rate-x))
+                (fee (mul-up dx (get fee-rate-x pool)))
+                (fee-rebate (mul-down fee (get fee-rebate pool)))
                 (dx-net-fees (if (<= dx fee) u0 (- dx fee)))
                 (dy (try! (get-y-given-x token collateral expiry dx-net-fees)))
 
                 (pool-updated
                     (merge pool
                         {
-                            balance-x: (+ balance-x dx-net-fees),
+                            balance-x: (+ (+ balance-x dx-net-fees) fee-rebate),
```

Comment:
```suggestion
                            balance-x: (+ balance-x dx-net-fees fee-rebate),
```
Clarity addition accepts 2 or more arguments, therefore this one can be coded as only one operation.